kentikapi/internal/cloud/export.go

```diff
@@ -207,7 +211,7 @@ func cePayloadWithProperties(payload *cloudexportpb.CloudExport, ce *cloud.Expor
 	case ibmProvider:
 		payload.Properties = ibmPropertiesToPayload(ce)
 	default:
-		return nil, kentikerrors.New(kentikerrors.InvalidRequest, fmt.Sprintf("invalid cloud provider: %v", ce.Provider))
+		return nil, fmt.Errorf("invalid cloud provider: %v", ce.Provider)
```

Comment:
As the list of supported providers may grow and cause error, maybe "unsupported cloud provider" woud better describe the case

Reply:
Done: d5528a9